.github/CONTRIBUTING.md

```diff
@@ -734,7 +734,7 @@ Math operators like +, -, /, *, etc are up in the air, just choose which version
 #### Use
 * Bitwise AND - '&'
 	* Should be written as `variable & CONSTANT` NEVER `CONSTANT & variable`. Both are valid, but the latter is confusing and nonstandard.
-* Associated lists declarations must have their key value quoted if it's a string
+* Associated lists declarations should have their key value quoted if it's a string
```

Comment:
It must, or it won't work properly

Reply:
not true, this is because it WILL work, dm assumes that if its not a define or a var that what you are declaring as a associative key must be a string, so if you meant to make it a var or define but you misspell it then byond will not tell you
![Screenshot_1085](https://user-images.githubusercontent.com/15794172/126569115-6de63be3-f9f7-4199-b198-103b43fbb3f9.png)

Reply:
if you use this to remove gaslist typepaths I will rain hellfire on you